service/controller/v3/key/key.go

```diff
@@ -53,6 +53,10 @@ func HostPrivateNetwork(customObject v1alpha1.FlannelConfig) string {
 	return customObject.Spec.Bridge.Spec.PrivateNetwork
 }
 
+func IsInDeletionState(customObject v1alpha1.FlannelConfig) bool {
+	return customObject.GetDeletionTimestamp() != nil
+}
```

Comment:
I would call it `IsDeleted` to shorten the name, and that's what actually happen.

Reply:
Done.